src/handlers/script/types.ts

```diff
@@ -11,3 +11,8 @@ export enum ScriptHandler {
   Check = 'script:check',
   RunFromGenerator = 'script:run-from-generator',
 }
+
+export interface OpenScriptResult {
+  script: string
+  isExternal: boolean
+}
```

Comment:
In the implementation of #839, this interface will contain data to determine whether the script is a browser script or not. It could also contain the list of scenarios in the test e.g. to allow users to pick a specific scenario to validate.